internal/label/pullrequests.go

```diff
@@ -9,114 +9,74 @@ import (
-			lowestNumber := 0
-			opts.ListOptions.Page++
-			owner := parts[0]
-			repo := parts[1]
-			pullRequests, resp, err := githubClient.PullRequests.List(context.TODO(), owner, repo, opts)
-			if err != nil {
-				return fmt.Errorf("error listing pull requests: %w", err)
+
+		lowestNumber := 0
+
+		for _, pullRequest := range pullRequests {
+			lowestNumber = *pullRequest.Number
+			if *pullRequest.Number < fullRepo.MinimumNumber {
+				// Break, not continue, since our order ensures PR numbers are getting smaller
+				break
+			}
+			if *pullRequest.Draft {
+				continue
+			}
+			user := *pullRequest.User.Login
+			if cfg.LabelSkipMap[user] {
+				continue
+			}
+			var label string
+			if teamMembers[user] {
+				label = cfg.LabelInternal
+			} else {
+				label = cfg.LabelExternal
 			}
 
-			for _, pullRequest := range pullRequests {
-				lowestNumber = *pullRequest.Number
-				if *pullRequest.Number < fullRepo.MinimumNumber {
-					// Break, not continue, since our order ensures PR numbers are getting smaller
-					break
-				}
-				if *pullRequest.Draft {
-					continue
-				}
-				user := *pullRequest.User.Login
-				if cfg.LabelSkipMap[user] {
-					continue
-				}
-				var label string
-				if teamMembers[user] {
-					label = cfg.LabelInternal
-				} else {
-					label = cfg.LabelExternal
+			if label != "" {
+				log.Printf("Pull Request %d by %s will be labeled %s\n", *pullRequest.Number, user, label)
+				hasLabel := false
+				for _, existingLabel := range pullRequest.Labels {
+					if *existingLabel.Name == label {
+						log.Println("  Already labeled, skipping...")
+						hasLabel = true
+						break
+					}
 				}
 
-				if label != "" {
-					log.Printf("Pull Request %d by %s will be labeled %s\n", *pullRequest.Number, user, label)
-					hasLabel := false
-					for _, existingLabel := range pullRequest.Labels {
-						if *existingLabel.Name == label {
-							log.Println("  Already labeled, skipping...")
-							hasLabel = true
-							break
-						}
+				if !hasLabel {
+					allLabels := []string{label}
+					for _, labelP := range pullRequest.Labels {
+						allLabels = append(allLabels, *labelP.Name)
 					}
-
-					if !hasLabel {
-						allLabels := []string{label}
-						for _, labelP := range pullRequest.Labels {
-							allLabels = append(allLabels, *labelP.Name)
-						}
-						_, _, err := githubClient.Issues.AddLabelsToIssue(context.TODO(), owner, repo, *pullRequest.Number, allLabels)
-						if err != nil {
-							return fmt.Errorf("error adding labels to pull request %d: %w", *pullRequest.Number, err)
-						}
+					_, _, err := githubClient.Issues.AddLabelsToIssue(context.TODO(), owner, repo, *pullRequest.Number, allLabels)
+					if err != nil {
+						return fmt.Errorf("error adding labels to pull request %d: %w", *pullRequest.Number, err)
 					}
 				}
 			}
+		}
 
-			if resp.NextPage == 0 || lowestNumber <= fullRepo.MinimumNumber {
-				break
-			}
+		if lowestNumber <= fullRepo.MinimumNumber {
+			break
```

Comment:
In this case, I think we just need to `continue` instead of `break` so that we can look at the other PRs